php/README.md

```diff
@@ -0,0 +1,73 @@
+# HubSpot-php sample Timeline app
```

Comment:
Should the main README be in the top level, but this one contain only lang specific instructions?

Reply:
readmes will be fixed in separate pr